src/parser/parsers/attributes.py

```diff
@@ -7,7 +7,10 @@
 import utils.json_utils as json_utils
 import maps as maps
 
-
+"""
+Output-data is used by https://deadlocked.wiki/Template:StatBoxes
+to display a hero's attributes on their hero page
+"""
```

Comment:
Sorry to be annoying, this should go below the class to be included in the docstring. I know I've been doing it wrong in some places, but I'm finding myfeet 😅

Reply:
ohhh I see, that makes sense actually. Done!